src/clj/fluree/db/json_ld/credential.cljc

```diff
@@ -113,14 +113,17 @@
        {:subject subject :did auth-did}))))
 
 (defn verify-jws
-  [jws]
+  [jws format]
   (let [{:keys [payload pubkey]} (or (crypto/verify-jws jws)
                                      (throw (ex-info (str "Invalid JWS: " jws)
                                                      {:status 400
                                                       :error  :db/invalid-credential})))
         id                       (crypto/account-id-from-public pubkey)
-        auth-did                 (did/auth-id->did id)]
-    {:subject (json/parse payload false) :did auth-did}))
+        auth-did                 (did/auth-id->did id)
+        processed-payload        (if (= :sparql format)
+                                   payload
+                                   (json/parse payload false))]
```

Comment:
Per the alternative approach taken in https://github.com/fluree/db/pull/843 we should not be parsing JSON in the function at all.